src/util.h

```diff
@@ -226,6 +234,20 @@ bool SoftSetBoolArg(const std::string& strArg, bool fValue);
 
 // Forces a arg setting, used only in testing
 void ForceSetArg(const std::string& strArg, const std::string& strValue);
+};
+
+extern ArgsManager argsGlobal;
```

Comment:
I was thinking that maybe some functions could get a local ```ArgsManager& args``` parameters instead of using the global. Or do you mean s/ArgsManager/Args/ ? I'm fine with that.
Also, maybe s/argsGlobal/gArgs/ too, but I don't like ```extern ArgsManager args```